src/app/pages/your-virtrolio/your-virtrolio.component.html

```diff
@@ -90,9 +91,17 @@ <h5 class="modal-title" id="send-message-modal-title">You're barking up the wron
       </div>
 
       <!-- Modal Body -->
-      <div class="modal-body" id='send-message-body'>
-        To sign a friend's virtrolio, they need to give you their 'sharing link'! To access <em>your</em>
-        sharing link, click the 'Sharing Link' button on this page!
+      <div class="modal-body sharing-link-body">
+        <p>
+          To sign a friend's virtrolio, they need to give you their 'sharing link'! You can plug in your friend's sharing
+          link below or directly into your browser search bar. To access <em>your</em> sharing link, click the 'Sharing Link' button on this page!
+        </p>
+        <label id="input-sharing-label" for="input-sharing-field" style="display:none">Visit friend link</label>
+        <input type="text" class="align-middle" #inputSharing value="Input your friend's link here"
+               (click)="copyLink(inputSharing)" id="input-sharing-field" [(ngModel)]='visitLink'>
+        <button type="button" id="button-copy" class="btn btn-primary align-middle" data-dismiss="modal" aria-label="Close"
+                (click)="navigateToLink()">Let's go
```

Comment:
Unnecessary wordiness
```suggestion
                (click)="navigateToLink()">Go
```